UsingTheServer.md

```diff
@@ -0,0 +1,155 @@
+# Using Your Own Application
+
+The server is currently set up to run with a default cobspec application.  This guide will help you set up the server for use with your own application.
+
+## Adding a module
+
+Much like the cobspec application, your application should be added as a module to the project structure.
+
+httpServer    
+|_ cobspec      
+|_ server   
+|_ YOUR_APPLICATION
```

Comment:
seems weird to have to clone this project and build in the directory and modify the existing code, I'd rather have controller over my app without worrying about breaking anything. Is there anyway I can just use my own Main, and if I'm using gradle add:

```
dependencies {
    compile group: 'com.saralein', name: 'server', version: '{latest}'
}
```
?